applications/RomApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -50,6 +50,8 @@ void AddCustomUtilitiesToPython(pybind11::module& m)
         .def_static("SetHRomComputingModelPart", &RomAuxiliaryUtilities::SetHRomComputingModelPart)
         .def_static("SetHRomVolumetricVisualizationModelPart", &RomAuxiliaryUtilities::SetHRomVolumetricVisualizationModelPart)
         .def_static("GetHRomConditionParentsIds", &RomAuxiliaryUtilities::GetHRomConditionParentsIds)
+        .def_static("AddModelPartConditionsToHROM", &RomAuxiliaryUtilities::AddModelPartConditionsToHROM)
+        .def_static("AddModelPartElementsToHROM", &RomAuxiliaryUtilities::AddModelPartElementsToHROM)
```

Comment:
What are these functions doing exactly?

- If, as their name suggests, they ADD either elements or conditions to a list, then why shall we expand manually the list after applying the function? 
- If these functions were called GetIds, the list should not be modified inside the functions. And then, modifying the list after appying the function based on the ids returned, would be the expected behaviour.